packages/utilities/fast-web-utilities/package.json

```diff
@@ -70,24 +70,21 @@
     ]
   },
   "devDependencies": {
-    "@babel/core": "^7.0.0",
-    "@babel/preset-env": "^7.1.0",
+    "@babel/core": "^7.12.13",
+    "@babel/preset-env": "^7.12.13",
     "@microsoft/eslint-config-fast-dna": "^1.2.0",
     "@types/jest": "^25.2.1",
-    "@types/lodash-es": "^4.17.0",
+    "@types/lodash-es": "^4.17.4",
     "babel-core": "^7.0.0-bridge.0",
     "eslint-config-prettier": "^6.10.1",
     "eslint-loader": "^4.0.0",
     "jest": "^25.4.0",
-    "lodash-es": "^4.17.10",
+    "lodash-es": "4.17.15",
     "prettier": "2.0.2",
     "ts-jest": "^25.4.0",
     "ts-loader": "^4.0.1",
     "typescript": "^3.9.0"
   },
-  "peerDependencies": {
```

Comment:
Can you explain the rational for removing the peer dependency? Won't this cause "module not found" error if the author uses this package but doesn't explicitly install `lodash-es`? I understand they need to do that either way, but w/ peerDependencies they get a warning that they need to install `lodash-es`

Reply:
This should simply act as a notification that the dependency is needed. If we remove this I don't think we get any benefit in our build, but it does make things more difficult in others' builds.